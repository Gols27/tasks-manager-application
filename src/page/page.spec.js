import React from 'react';
import Page from './';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

const props= { 
  toggleStatus : jest.fn(),
  item: { 
    id : '123', 
    name: '1234', 
    status: 'Pending',
  }
}

it('renders without crashing', () => {
  const wrapper = shallow(<Page {...props}/>);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
