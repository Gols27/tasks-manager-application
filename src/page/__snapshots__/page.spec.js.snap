// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="page Pending"
>
  <div
    className="page-content"
    onClick={[Function]}
  >
    <div>
      1234
    </div>
    <div>
      Pending
    </div>
  </div>
</div>
`;
